agent/properties/MANIFEST.MF

```diff
@@ -1,3 +0,0 @@
-Class-Path: lib/ant-launcher.jar config/

```

Comment:
Any reason for this file to be removed. I started the Development Server and tried to access the agents page. Got the exception `Could not load compiled manifest from /Users/varshavs/gocd/server/webapp/WEB-INF/rails.new/public/assets/webpack/manifest.json - have you run `rake webpack:compile`? (original error No such file or directory - /Users/varshavs/gocd/server/webapp/WEB-INF/rails.new/public/assets/webpack/manifest.json)`